server/src/service.rs

```diff
@@ -206,8 +204,7 @@ impl Service {
         let object_server = connection.object_server();
         let service = Self {
             collections: Default::default(),
-            manager: Default::default(),
-            connection: connection.clone(),
+            manager: Arc::new(Mutex::new(ServiceManager::new(connection.clone()))),
```

Comment:
Why do you need to clone the connection here? nothing uses it afterwards

Reply:
If I'm to loose the clone here, I'd have to clone the connection above,
```let object_server = connection.clone().object_server();```

Otherwise, `cannot move out of `connection` because it is borrowed`.

Should I apply the recommendation made by the compiler?

Reply:
Or keep this as it is?

Reply:
It doesn't work either (compiler recommendation). To remove the clone here I'll have to do: `let object_server = connection.object_server().clone();`